src/app/pages/loading-states/loading-states.component.scss

```diff
@@ -0,0 +1,39 @@
+@import '~@pxblue/colors/palette.scss';
+
```

Comment:
We should be able to eliminate some of these rules if we use InfoListItem

Reply:
You're right, it did